xmodule/js/src/html/edit.js

```diff
@@ -1410,7 +1410,7 @@
       if (this.editor_choice === 'visual') {
         visualEditor = this.getVisualEditor();
         raw_content = visualEditor.getContent({
-          format: "text",
+          format: "raw",
```

Comment:
This reverts a change in https://github.com/openedx/edx-platform/pull/30335/files#diff-bd3e3877ed43b0228f0227205afd052479229747ee194918aab121957b9b0edbR1393 

@DubeySandeep @Agrendalath I don't suppose you could tell us why you made this change when upgrade TinyMCE to v5.5.1?

Reply:
@pdpinch, TinyMCE docs [replaced](https://github.com/openedx/edx-platform/pull/30335#discussion_r877696847) all occurrences of the `raw` format with `text` between v4 and v5. These formats are undocumented, and we mistakenly assumed that this is the new way to get/set content in TinyMCE v5.

cc: @tecoholic

Reply:
@pdpinch What @Agrendalath mentioned in the previous comment. I made the change thinking that the TinyMCE has changed the values from raw to text as there were no references to `raw` as a value in the docs.